test/addons-abi/test_buffer/test_buffer.cc

```diff
@@ -0,0 +1,141 @@
+#include <string.h>
+#include <string>
+#include <node_api_helpers.h>
+
+#define JS_ASSERT(env, assertion, message) \
```

Comment:
This macro looks useful! Should it go in a header shared by all the tests? Maybe also the next macro.